reposilite-frontend/src/components/browser/Card.vue

```diff
@@ -225,4 +254,41 @@ export default {
 .token.string {
     color: mediumpurple;
 }
+.config-dropdown {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+  margin: 24px 0;
+}
+.config-dropdown ul {
+  display: none;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 10px;
+  background-color: #e5e5e56e;
+}
+.selected-config {
+  width: 100%;
+  padding: 5px 10px;
+  border-radius: 8px;
+  border: 1px solid #e5e5e5;
+  box-sizing: border-box;
+}
+.dropdown-icon {
+  width: 20px;
+  height: 25px;
+  float: right;
+  margin: auto;
+  display: flex;
+  align-items: center;
+}
```

Comment:
In general, with WindiCSS we don't really want to use raw CSS properties. Take a look at https://windicss.org/ to search for alternative properties, e.g. `flex-direction: column;` can be replaced with `flex-col`.

Reply:
Done. Replaced raw css with utility classes